### PR TITLE
readme styling, routes, and module ordering in index.html

### PR DESCRIPTION
**Pull request by azdoolittle**

### CODE REVIEW

**Comment by Nguyephi** [comment]
lgtm

File: README.md [hunk 1]
```diff
@@ -1,19 +1,26 @@
 # Andrew's Proxy
-This repo is a proxy for Twitch.tv combining the modules from the following projects
+This repo is a proxy for Twitch.tv combining the modules from the projects below
 
 ## Related Projects
```
Comment (priscilagonzalez) [inline README.md:4]: I like this (README)!

**Review by priscilagonzalez — COMMENTED** [review]
Lgtm.

File: server/index.js [hunk 2]
```diff
@@ -5,39 +5,39 @@ const port = 3000;
...
+  if (type in hosts) {
+    return fetch(`http://${req.hostname}:${hosts[type]}${req.url}`)
       .then((results) => results.json())
       .then((results) => res.send(results));
   };
 }
 
 app.get('/api/player/:Id', (req, res) => {
-  return handleAPI('player', req, res);
+  return handleHosts('player', req, res);
```
Comment (kraig-droid) [inline server/index.js:24]: Hmm... I think handle API was better

**Review by kraig-droid — APPROVED** [review]

**Pull request merged**